apps/gitness/src/pages-v2/pull-request/pull-request-conversation.tsx

```diff
@@ -440,10 +440,14 @@ export default function PullRequestConversationPage() {
       head_branch: pullReqMetadata?.source_branch,
       head_commit_sha: pullReqMetadata?.source_sha
     }
-    rebaseBranch({ body: payload, repo_ref: repoRef }).then(() => {
-      onPRStateChanged()
-      setRuleViolationArr(undefined)
-    })
+    rebaseBranch({ body: payload, repo_ref: repoRef })
+      .then(() => {
+        onPRStateChanged()
+        setRuleViolationArr(undefined)
+      })
+      .catch(error => {
```

Comment:
i'd prefer if we use the rejection callback of `.then()` instead of `catch()`. 
catch triggers for JS exceptions as well, preventing them from bubbling up and swallowing errors.